### PR TITLE
DNS MX Record misconfiguration

### PR DESCRIPTION
**Pull request by ohachimOs**
This PR adds **DNS MX Record misconfiguration** KB.

### CODE REVIEW

File: WEB_SERVICE/WEB/_LOW/DNS_MX_RECORD_MISCONFIGURATION/description.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# MX Record Misconfiguration
+
+MX (Mail Exchanger) records in DNS are critical for email routing and delivery. Misconfigurations in MX records can lead to email delivery failures, increased vulnerability to email spoofing, and inefficient mail routing. The following areas are key concerns in MX record configuration:
+
+1. **Record Format**: MX records must adhere to the standard format of 'priority hostname.domain.tld'. Incorrect formatting can lead to misinterpretation by email servers and DNS resolvers. Priority values should be valid integers, and hostnames must be properly formatted.
+
+2. **Priority Values**: MX record priorities should be unique and within the valid range of 0-65535. The lowest value indicates the most preferred mail server. Improperly ordered or invalid priorities can cause inefficient email routing.
```
Comment (Zakaria-Latif) [inline WEB_SERVICE/WEB/_LOW/DNS_MX_RECORD_MISCONFIGURATION/description.md:7]: multiple servers can share the same priority value for load balancing, that uniqueness applies only when you don't intend load balancing

**Review by Zakaria-Latif — CHANGES_REQUESTED** [review]

File: WEB_SERVICE/WEB/_LOW/DNS_MX_RECORD_MISCONFIGURATION/description.md (hunk 1 above)
Comment (ohachimOs) [reply WEB_SERVICE/WEB/_LOW/DNS_MX_RECORD_MISCONFIGURATION/description.md:7]: Got it

File: WEB_SERVICE/WEB/_LOW/DNS_MX_RECORD_MISCONFIGURATION/meta.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "REQ_1_3",
+        "REQ_4_1",
+        "REQ_12_2"
+      ],
+      "SOC2_CONTROLS": [
+        "CC_4_1",
+        "CC_6_1",
+        "CC_6_6",
+        "CC_7_1",
+        "CC_7_2"
```
Comment (nmasdoufi-ol) [inline WEB_SERVICE/WEB/_LOW/DNS_MX_RECORD_MISCONFIGURATION/meta.json:22]: Please check if: GDPR Compliance is applicable in this case, especially:

- Article 32 (Security of Processing)
- Article 25 (Data Protection by Design and by Default)

File: WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/description.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# MX Record Misconfiguration
```
Comment (3asm) [inline WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/description.md:1]: No title.

File: WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/recommendation.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+To address MX record misconfigurations, implement the following:
```
Comment (3asm) [inline WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/recommendation.md:1]: Give examples please.

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by nmasdoufi-ol — APPROVED** [review]

File: WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/recommendation.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+* **Correct Record Format**: Ensure all MX records follow the exact 'priority hostname.domain.tld' format. Example:
```
Comment (ErebusZ) [inline WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/recommendation.md:3]:
```suggestion
* **Correct Record Format**: Ensure all MX records follow the exact 'priority hostname.domain.tld' format.
Example:
```

Comment (ErebusZ) [reply WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/recommendation.md:3]: apply to the rest (if you agree)

Comment (ohachimOs) [reply WEB_SERVICE/DNS/_MEDIUM/DNS_MX_RECORD_MISCONFIGURATION/recommendation.md]: image

**Review by 3asm — APPROVED** [review]

**Pull request merged**